automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/methods/MethodConfigDetailsPage.scala

```diff
@@ -216,9 +219,9 @@ class LaunchAnalysisModal(implicit webDriver: WebDriver) extends FireCloudView {
       click on (await enabled launchAnalysisButtonQuery)
     }
 
-    def getSelectedEntityText(): String = {
-      await enabled selectedEntityQuery
-      find(selectedEntityQuery).fold("")(_.text)
+    def isNoDefaultEntitiesMessagePresent(): Boolean = {
+      await enabled emptyDefaultEntitiesMessageQuery
```

Comment:
Probably don't need to wait for it to be enabled and also find it. Could probably just find (or findAll if you expect to have to wait... see Scalatest or Selenium docs... I don't have a good link handy).

Reply:
i needed the wait there.

Reply:
Right, sorry... that was me reviewing from my phone in front of Starbucks on eclipse day. I guess maybe it's the `find(...)` below that you don't technically need. All depends on how you look at it. I'm fine with it either way.